kong/tools/ciphers.lua

```diff
@@ -0,0 +1,114 @@
+local cat = table.concat
```

Comment:
We try to avoid such globals caching when not necessary (cold paths) - this looks like such a case, since this is only to be called by the CLI.